pkg/restapi/operation/operations_test.go

```diff
@@ -31,7 +33,7 @@ import (
 )
 
 const (
-	testVaultID = "urn:uuid:abc5a436-21f9-4b4c-857d-1f5569b2600d"
+	testReferenceID = "urn:uuid:abc5a436-21f9-4b4c-857d-1f5569b2600d"
```

Comment:
Also, just to make it clear that the reference ID doesn't have to be a UUID, it would probably be best to change this value to something else...anything else, really. maybe just "testReferenceID".

Reply:
fixed